packages/@uppy/image-editor/src/ImageEditor.tsx

```diff
@@ -0,0 +1,229 @@
+import { UIPlugin, type UIPluginOptions, type Uppy } from '@uppy/core'
+import type { DefinePluginOpts } from '@uppy/core/lib/BasePlugin.js'
+import type Cropper from 'cropperjs'
+import { h } from 'preact'
+
+import type { Meta, Body, UppyFile } from '@uppy/utils/lib/UppyFile'
+import Editor from './Editor.tsx' // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
nit: That comments seems unrelated with this line
```suggestion
import Editor from './Editor.tsx'
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```